### ISSUE TITLE
[Bug]: Problem with download_sentinel_orbits_linux.csh due to migration of Open Access Hub

### ISSUE THREAD

**Issue by Tijn-Nederstigt** [gmtsar/gmtsar#853]
**Problem**
The script download_sentinel_orbits_linux.csh is not able to retrieve orbit files anymore for Sentinel-1A data. I wonder whether other users have encountered this issue with this or related scripts, and whether this has to due with  the recently announced move from of the Copernicus Open Access Hub to the Copernicus Data Space Ecosystem (https://scihub.copernicus.eu/news/News01242)

Is a fix for this upcoming?

**Input**
My SAFE_filelist is as follows (although I had no success with several other SAFE_filelists, whereas this used to be no problem before:
/mnt/d/Projecten/Bodemdaling/Working_directories/Mijnwater/asc/data/S1A_IW_SLC__1SDV_20231105T172507_20231105T172534_051085_062905_7D23.SAFE

My command is as follows:
download_sentinel_orbits_linux.csh SAFE_filelist 1

**System and software version:**
 - OS: Ubuntu-20.04 on WSL [Windows 10]
 - GMTSAR version: 5.7

**Processing Log**
Below is my WSL output.

Downloading Precise Orbits (POEORB)...

---------------------
Finding orbits for S1A_IW_SLC__1SDV_20231105T172507_20231105T172534_051085_062905_7D23.SAFE...
Required orbit file dates: 20231104 to  20231106...
2023-11-04T00:00:00.000Z
2023-11-06T23:59:59.999Z
Querying ESA POD Hub archive...
--2023-11-23 15:01:38--  https://scihub.copernicus.eu/gnss/search?q=beginPosition:[2023-11-04T00:00:00.000Z%20TO%202023-11-06T23:59:59.999Z]%20AND%20endPosition:[2023-11-04T00:00:00.000Z%20TO%202023-11-06T23:59:59.999Z]%20AND%20platformname:Sentinel-1%20AND%20filename:S1A_*%20AND%20producttype:AUX_POEORB
Resolving scihub.copernicus.eu (scihub.copernicus.eu)... 146.59.215.226
Connecting to scihub.copernicus.eu (scihub.copernicus.eu)|146.59.215.226|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://scihub.copernicus.eu/maintenance.html [following]
--2023-11-23 15:01:39--  https://scihub.copernicus.eu/maintenance.html
Reusing existing connection to scihub.copernicus.eu:443.
HTTP request sent, awaiting response... 200 OK
Length: 828 [text/html]
Saving to: ‘orbitquery.txt’

orbitquery.txt                100%[=================================================>]     828  --.-KB/s    in 0s

2023-11-23 15:01:39 (101 MB/s) - ‘orbitquery.txt’ saved [828/828]

Checking query for existing orbit file...
Query Failed -- possible issues:
 (1) couldn't connect to ESA POD hub (check manually)
 (2) an orbit file for those dates does not exist yet
     -- check the resistited orbit files`

**Comment by colinintherain** [comment]
yeah,I have encountered this issue yesterday, I  downloaded the orbits data from the Copernicus Data Space Ecosystem manually

**Comment by todowede** [comment]
Had the same issue this morning. got the orbit data from Copernicus

**Comment by Xiaohua-Eric-Xu** [comment]
Please see this discussion page for the issue: https://github.com/gmtsar/gmtsar/discussions/852

**Comment by Xiaohua-Eric-Xu** [comment]
Temporarily added two scripts to download orbits from https://step.esa.int/auxdata/orbits/Sentinel-1. https://github.com/gmtsar/gmtsar/commit/8df3cc255e116ea3f52181cf4941f1becbf14eac

**Comment by Tijn-Nederstigt** [comment]
Many thanks. I will try these out!

**Comment by mobigroup** [comment]
You can use PyGMTSAR ASF module to download required subswaths and polarizations only and the corresponding orbits from Alaska Satellite Facility. That’s so hard and useless to download full Sentinel-1 scenes! While it requires ASF user registration, the module downloads the complete dataset fast and requires much less disk space to store the separate subswaths only. It allows to process 30-60 scenes and hundreds of interferograms even on free Google Colab instances in some hours. Here is the example (not the largest one):

[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1J5E9KQnrZneiRf-wv2fDBCnEKh1m7kbK?usp=sharing) Gastein Valley, Austria, 2021—2023. PS-weighted SBAS and PSI example for 28 scenes and 201 interferograms.

<img src="https://github.com/mobigroup/gmtsar/assets/7342379/27bd0436-db6f-45cd-88c8-99332ec337d5" width="50%"><img src="https://github.com/mobigroup/gmtsar/assets/7342379/6d1181fb-5bdb-4923-ae64-74f59b48f9b4" width="50%">
